.gitignore

```diff
@@ -1,10 +1,15 @@
-/target/
-/.idea/libraries/
-/.idea/cssxfire.xml
-/.idea/workspace.xml
-.DS_Store
-/.idea/
-/hivemq4-prometheus-extension.iml
+# Gradle
+.gradle
+build/
 
+# IntelliJ
+out/
 
-dependency-reduced-pom.xml
+.idea/*
+!.idea/codeStyles
+!.idea/inspectionProfiles
+!.idea/runConfigurations
+.java-version
+.DS_Store
+
+/hivemq4-prometheus-extension.iml
```

Comment:
please use the same .gitignore as in other projects
```
# Gradle
.gradle
build/

# IntelliJ
out/
*.iml
.idea/*
!.idea/codeStyles
!.idea/inspectionProfiles
!.idea/runConfigurations

.java-version
.DS_Store
```

Reply:
done